sycl/ReleaseNotes.md

```diff
@@ -763,7 +763,7 @@ Release notes for the commit range ba404be..24726df
     BEs to perform additional optimizations [fdcaeae] [08f8656]
   - Added partial support for [Host task with interop capabilities extension](https://github.com/codeplaysoftware/standards-proposals/blob/master/host_task/host_task.md) [ae3fd5c]
   - Added support for [SYCL_INTEL_bitcast](doc/extensions/Bitcast/SYCL_INTEL_bitcast.asciidoc)
-    as a `sycl::detail::bit_cast` [e3da4ef]
+    as a `sycl::bit_cast` [e3da4ef]
```

Comment:
```suggestion
    as a `sycl::detail::bit_cast` [e3da4ef]
```

Reply:
I'm not sure that we should change this at all. These are for past releases: there is even commit hashes specified at line 758

Reply:
This made me wonder why we still have the  draft `SYCL_INTEL_bitcast` extension documentation at all...  We never implemented it (because what we did implement was in `sycl::detail::`), and what we're implementing now is not that extension but SYCL 2020.  Can we remove it?

Would be interested to hear from @pvchupin and other @intel/dpcpp-specification-reviewers on this.

Reply:
restored. thanks